aws_xray_sdk/ext/sqlalchemy_core/patch.py

```diff
@@ -48,7 +52,15 @@ def _sql_meta(instance, args):
 
 
 def _xray_traced_sqlalchemy_execute(wrapped, instance, args, kwargs):
-    name, sql = _sql_meta(instance, args)
+    return _fetch_sql_metadata_and_process_request(wrapped, instance, args, kwargs)
+
+
+def _xray_traced_sqlalchemy_session(wrapped, instance, args, kwargs):
+    return _fetch_sql_metadata_and_process_request(wrapped, instance.bind, args, kwargs)
+
+
+def _fetch_sql_metadata_and_process_request(wrapped, engine_instance, args, kwargs):
```

Comment:
What does the `fetch` mean? Is it fetching?

Reply:
By fetching if you mean getting info from some remote endpoint/service, then the function is not really _fetching_ any metadata rather it's just extracting info from db engine. Probably a better name can be `_populate_sql_metadata_and_process_request` or simply `_process_request`?

Reply:
renamed